src/test/java/no/digipost/concurrent/OneTimeToggleTest.java

```diff
@@ -27,19 +40,60 @@
 
     @Test
     public void toggleIsAllowedSeveralTimes() {
-        assertThat(done.yet(), is(false));
+        assertThat(done, whereNot(OneTimeToggle::yet));
         done.now();
-        assertThat(done.yet(), is(true));
+        assertThat(done, where(OneTimeToggle::yet));
         done.now();
-        assertThat(done.yet(), is(true));
+        assertThat(done, where(OneTimeToggle::yet));
```

Comment:
Hvorfor endrer vi dette? Det forrige leste bedre synes jeg, hvis funksjonaliteten er den samme?🤷‍♂️

Reply:
Sant. Leser kanskje ikke helt sånn optimalt. Ville kanskje lest litt bedre med sånn POJO-style en `.isToggled()`-metode i stedet for den dødshippe og innovative `.yet()`.

Forskjellen ligger i feilmeldingen. Feilmelding dersom testen feiler med endret kode:
> Expected: OneTimeToggle with a yet which is `<true>`
>     but: had the yet `<false>`

vs. feilmelding med forrige kode:
> Expected; is `<true>`
>     but: was `<false>`

Reply:
Hehe tok litt tid før jeg skjønte hva `.yet()` var ja😅 Men da forstår jeg, feilmeldingen er jo mye bedre nå😊

Reply:
The Abominable Snowman https://en.wikipedia.org/wiki/Yeti#The_%22Abominable_Snowman%22